### ISSUE TITLE
Remove limited servers list from plain view

### ISSUE THREAD

**Issue by ringtailsoftware** [mastodon/mastodon#20326]
### Pitch

In `/about/more#unavailable-content` every server lists the limited servers. This transparency is welcome, but it also means that every Mastodon server explicitly links to some of the worst places imaginable online.

I do not have a good solution which preserves the transparency, but perhaps this could be hidden behind a CAPTCHA, or made available by email request?

### Motivation

For a person looking for such content, this list a goldmine. Search engines may also pick up these links and de-rank accordingly.

**Comment by trwnh** [comment]
pretty sure it's an admin setting to show to everyone / logged-in users / no one, but in any case they aren't actually links, they're just plaintext domains

**Comment by ClearlyClaire** [comment]
Yes, this is controlled by an admin setting, this information can be made available to either:
- nobody
- logged-in users only
- everyone

Individual domains can also be somewhat obfuscated (this is a per-domain-block option that automatically replaces part of the domain with asterisks).

Closing, but feel free to reopen if you don't think this answers your concerns.
